server/sonar-web/src/main/js/apps/quality-profiles/views/ChangeProjectsView.js

```diff
@@ -27,6 +28,8 @@ export default ModalFormView.extend({
   template: Template,
 
   onRender() {
+    // TODO remove uuid usage
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1135)